mne/io/edf/edf.py

```diff
@@ -613,7 +613,7 @@ def _read_edf_header(fname, exclude):
         for i, unit in enumerate(units):
             if i in exclude:
                 continue
-            if unit == 'uV':
+            if unit in ('uV', 'µV', 'µV'):
```

Comment:
why 2 times µV ?

besides LGTM

Reply:
as @cbrnr suggested that there are two special symbols for microvolt as per the wiki page https://en.wikipedia.org/wiki/Micro-#Symbol_encoding_in_character_sets the greek mu symbol and another legacy micro symbol used to denote the same. Just as a precaution I have implemented both for units to denote data in microvolts.

Reply:
they're supposed to be two different codepoints (μ `03BC` greek small letter mu; µ `00B5` micro sign) but copy/paste from the GH website suggests they're the same (both micro sign).  I suggest either:

1. make sure they're actually different codepoints and add a comment to this line explaining why it looks redundant, or
2. change them to `'\u03BCV'` and `\u00B5V'` and add a comment explaining why using codepoints

Reply:
@drammock I think something like this  `if unit in ('\u03BCV','\u00B5V','uV'):` might work then, what should I do now to make this change

Reply:
you'll need to make a new PR based off of current `main`. Pushing more commits to this branch won't work because it's already merged.